com.unity.render-pipelines.universal/Editor/StandardRendererDataEditor.cs

```diff
@@ -5,12 +5,12 @@
 
 namespace UnityEditor.Rendering.Universal
 {
-    [CustomEditor(typeof(ForwardRendererData), true)]
-    [MovedFrom("UnityEditor.Rendering.LWRP")] public class ForwardRendererDataEditor : ScriptableRendererDataEditor
+    [CustomEditor(typeof(StandardRendererData), true)]
+    [MovedFrom("UnityEditor.Rendering.LWRP")] public class StandardRendererDataEditor : ScriptableRendererDataEditor
```

Comment:
The type that has been moved from LWRP is `ForwardRendererDataEditor` whence you should remove the `MovedFrom` attribute here

Reply:
Moved back to `ForwardRendererDataEditor `.